.changes/v0.13.1.md

```diff
@@ -0,0 +1,20 @@
+## v0.13.1 - 2024-09-18
+
+
+### Added
+- Added new methods Container.withoutFiles and Directory.withoutFiles by @jedevc in https://github.com/dagger/dagger/pull/8216
```

Comment:
Oops, I've tried to be consistent with using backticks in these, I think I missed that one :cry:

Reply:
no problem, fixed!